### PR TITLE
Toggle

### PR DESCRIPTION
**Pull request by jmc617**

### CODE REVIEW

File: src/components/Player/Opsheet/List.js [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-            <li>Four</li>
-            <li>Five</li>
-          </ul>
-        </div>
-      );
-    }
-  }
+import React from 'react';
 
+function List(props) {
```
Comment (dduleone) [inline src/components/Player/Opsheet/List.js]: If you'd like this to be a "stateless component" I'd recommend following the convention of:

```javascript
const List = (props) => {
    // ...
}
```

File: src/components/Player/Opsheet/List.js [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
+  return <div className={props.listClassName}  >
```
Comment (dduleone) [inline src/components/Player/Opsheet/List.js]: Also, I highly recommend getting in the habit of destructuring your variables, instead of using dot notation, when possible. It makes code easier to type, easier to read for other people, and also more reusable.

File: src/components/Player/Opsheet/Opsheet.js [hunk 3]
```diff
@@ -59,7 +59,7 @@ class Opsheet extends Component {
           <button className='collapse-toggle' onClick={this.toggleSheetCollapseExpand}>+/-</button>
           
           <ListType />
-          <List listClassName = {this.listClassName}/> 
+          <List listClassName={this.state.listClassName}/>
```
Comment (dduleone) [inline src/components/Player/Opsheet/Opsheet.js]: This illustrates one reason I like destructuring.

If you had destructured:
```JavaScript
const {listClassName} = this;
```
in the first place, you could now change it to:
```JavaScript
const {listClassName} = this.state;
```
and anywhere you used `listClassName` would instantly be supplied with the new alternate value. Alternatively, if you had used this variable in several places, you'd have to go through and change it in every place.

Plus, destructuring at the top of your method allows someone who's reading your code from the top to the bottom to see what values should be found on which objects, and therefore which objects should be expected to have which values... and confidence that they're coming off the correct variables.

in other words, if you had used `this.props.listClassName` I wouldn't know if that's because you intended this to be a prop, or if you typo'd and meant to take the value off `state`.

But using `const {listClassName} = this.state;` is very "declarative" and sets up whoever is reading your code to understand what preconditions are necessary to use your function - as well as focusing your efforts by first setting up what variables should be available to you, and not having to remember what properties should be found on which objects, when you come back in a week to continue hacking on it.
